experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -867,13 +867,9 @@ describe("SharedTree", () => {
 			const [tree1, tree2] = provider.trees;
 
 			// Insert node
```

Comment:
We are no longer inserting node. What happened here?

Reply:
Ah, I replaced it with the initialize tree so it's just a bit of cleanup. I'll modify the comments.